src/dp_vnf.c

```diff
@@ -27,6 +35,46 @@ void dp_vnf_free(void)
 	dp_free_jhash_table(vnf_handle_tbl);
 }
 
+static int dp_record_vnf_value(struct dp_vnf *vnf)
```

Comment:
This just may be my Slavic mind, but `record` sounds like a world record or music record and not "save" or "add" which this function does :)

Reply:
ja, I was hesitating to use record as well. so changed to "add'